src/control/positionControl/CollisionDetector.cpp

```diff
@@ -52,8 +52,8 @@ std::vector<Vector2> CollisionDetector::getRobotPositions() {
     return robotPositions;
 }
 
-void CollisionDetector::setField(const rtt::world::Field& field) { this->field = &field; }
+void CollisionDetector::setField(const rtt::world::Field& field_) { this->field = &field_; }
```

Comment:
This will give the same problem as above; very bug and segfault prone.

Reply:
Though it seems, this was not your fault as the previous team did this.

Reply:
Do you want me to change this? It is currently only used by the old pathplanning